src/main/resources/META-INF/rust-core.xml

```diff
@@ -1171,6 +1171,8 @@
                      description="Expand macros in parallel"/>
         <registryKey key="org.rust.external.linter.max.duration" defaultValue="3000" restartRequired="false"
                      description="Show notification warning if the external linter is running longer than the set value (ms)"/>
+        <registryKey key="org.rust.ide.json.paste.processor" defaultValue="false" restartRequired="false"
```

Comment:
Let's enable it in `RsConvertJsonToStructCopyPasteTest` to fix the corresponding tests